addons/exact-count/addon.json

```
@@ -29,6 +35,22 @@
       "if": {
         "settings": { "user": true }
       }
+    },
+    {
+      "url": "studio.js",
+      "matches": ["https://scratch.mit.edu/studios/*"],
```

Just to be consistent with other addons:

```suggestion
      "matches": ["studios"],
```